projects/plugins/jetpack/_inc/lib/class.core-rest-api-endpoints.php

```diff
@@ -3960,6 +3979,21 @@ public static function update_licensing_error( $request ) {
 		return true;
 	}
 
+	/**
+	 * Set a Jetpack license
+	 *
+	 * @since 9.5.0
+	 *
+	 * @param WP_REST_Request $request The request.
+	 *
+	 * @return WP_REST_Response A response object.
+	 */
+	public static function set_jetpack_license( $request ) {
+		Licensing::instance()->append_license( $request['license'] );
```

Comment:
`update_option` might fail to `append_license`. Not sure if we want to return a non-200 response in that case.